modules/dists/BlockDist.chpl

```diff
@@ -1767,35 +1772,42 @@ proc BlockArr.doiScan(op, dom) where (rank == 1) &&
       if debugBlockScan then
         writeln(locid, ": ", (numTasks, rngs, state, tot));
 
+      // Create a local ready var and store it back on the initiating
+      // locale so it can notify us
+      var myOutputReady = new unmanaged BoxedSync(bool)?;
```

Comment:
Oh, super-minor, but I'd also probably use `const` rather than `var` here if you change anything else...

Reply:
Yeah, I had similar thoughts for changing this to be non-nilable owned but ended up leaving it as-is for now because it matches the array type (which needs to be nilable and I think unmanaged?) and because we're still using unmanaged for the cloned ops.

I was wondering if those could be cleaned up too, but wanted to save that for a followup PR and keeps things as they are for now.

Reply:
Remind me -- what does `const` mean for Chapel classes? We won't change what we're pointing to, but the thing being pointed to can still be modified?

Reply:
Yeah, that's right.

Reply:
Made non-nilable in https://github.com/chapel-lang/chapel/pull/20185 (though stuck with unmanaged because having to do something like `outputReady[locid] = myOutputReady.borrow():unmanaged` didn't feel like an obvious elegance improvement. 

Switched from `var` to `const` in https://github.com/chapel-lang/chapel/pull/20100